testing/testcontainers/java/src/test/java/io/crate/example/testing/utils/TestingHelpers.java

```diff
@@ -36,4 +39,21 @@ public static void assertResults(Application.Results results) throws SQLExceptio
                 "Monte Rosa",
                 "Dom");
     }
+
+    /**
+     * Support function for initializing CrateDB with SQL using an init function.
+     * You can run a simple schema setup or Flyway/liquibase DB migrations here, at your disposal.
+     * <a href="https://www.testcontainers.org/modules/databases/jdbc/#using-an-init-function"/>
+     */
+    public static void sqlInitFunction(Connection connection) throws SQLException, IOException, InterruptedException {
+        try (Statement stmt = connection.createStatement()) {
+            boolean checkResults = stmt.execute("CREATE TABLE IF NOT EXISTS foobar_init (id INTEGER)");
+            if (checkResults) {
+                System.out.println("Success.");
```

Comment:
```suggestion
                System.out.println("DB initialized.");
```

Reply:
I think this is gone already, right?

Reply:
yep, since we don't check the boolean result, sorry but I commented before trying out the test.